xds/src/main/java/io/grpc/xds/XdsClientImpl.java

```diff
@@ -1387,8 +1387,6 @@ static Builder newBuilder() {
       private Node node;
       private com.google.rpc.Status errorDetail;
 
-      private Builder() {}
```

Comment:
Ah, I mean the whole Builder with setters are not necessary, can just use the constructor.

Reply:
I understand. Will do the removal in upcoming commit.